wiki/Beatmaps/Genre_and_language/en.md

```diff
@@ -0,0 +1,48 @@
+---
+tags:
+  - genres
+  - languages
+  - metadata
+---
+
+# Genre and language
+
+![A screenshot of Source, Genre, Language, and Tags on a beatmap page](img/beatmap-info.png "Genre and language are listed on beatmap pages.")
+
+[Beatmaps](/wiki/Beatmaps) submitted to the website have **genre** and **language** fields to help categorise songs. They default to `Unspecified` and `Other` respectively, and can be edited by mapset owners, [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) members, and [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) members.
```

Comment:
might want to add a detail which explains that once a map got Ranked/Loved the mapset owner loses their power to modify the Genre and Language settings in their beatmap

something like

> (...) and can be edited by [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) and [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) members. Furthermore, mapset owners are also able to set the genre and language of their own beatmaps as long as the said beatmap's status is either WIP, Pending, or Graveyarded.